portfolio/src/main/java/com/google/sps/data/Comment.java

```diff
@@ -0,0 +1,13 @@
+package com.google.sps.data;
+
+public class Comment {
+  public long id;
```

Comment:
Normally in Java, we use private for class attributes and create getter methods to access them.

Reply:
Converted Comment class to use private fields